core/admin/EE_Admin_Page.core.php

```diff
@@ -197,6 +197,19 @@ public function __construct($routing = true)
         $this->_req_data = array_merge($_GET, $_POST);
         // routing enabled?
         $this->_routing = $routing;
+    }
+
+
+    /**
```

Comment:
Seeing how we're introducing `initializePage`, it might be good to differentiate between it and `__construct()` in the comments (eg what goes in which; when should `initializePage` be called, etc). This will help make it clearer where new code should go if we need to add to them someday.
Also I'm a little nervous that this change might break something. I think we should go ahead with it, just try our best to test it, especially with add-ons.

Reply:
done